.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Create Docker image for release
```

Comment:
Not that this is part of this change.. but it got me thinking... we copy/paste this workflow around.  is there a way for us to create it once and have it be used everywhere... ?

found this https://docs.github.com/en/actions/using-workflows/reusing-workflows and thought u might find it interesting.

Reply:
I looked briefly into creating custom workflow earlier. I think it's a _really_ interesting option, I just haven't had a chance to try an MVP on it.

Definitely for things like "build docker image", "build deb", "submit to ECR", it could be interesting.

Reply:
cool. not gonna block this or anything for the this (re-use workflow)